index.html

```diff
@@ -42,10 +42,14 @@
                         <h1>We Take Care of the Future</h1>
                         <h4>We know how large objects will act, but things on a small scale just do not act that way.</h4>
                         
-                        <div class="buttons">
-                        <a href="" class="button button-get">Get Quote Now</a>
-                        <a href="" class="button button-learn">Learn More</a>
+                        <div class="buttons" id="big--buttons">
+                        <button class="button button--primary">Get Quote Now</button>
+                        <button class="button button--secondary">Learn More</button>
                         </div>
+                        <div class="buttons" id="MQ--buttons">
+                            <button class="button button--primary">Get</button>
+                            <button class="button button--secondary">Learn</button>
+                            </div>
```

Comment:
I think you need to delete this.

Reply:
Those buttons need for MQ, in small screen they look different and contain only "Get" "Learn" text.

Reply:
Well, just for fun who said that they look differently? @IvanCeban   :-)
![image](https://github.com/LiluUncharted/KMarketer/assets/67589913/a6e8137d-c264-4d3c-8830-29bba7e48a3d)

Reply:
Slack, home-works - look there)) not on Figma
![image](https://github.com/LiluUncharted/KMarketer/assets/141845795/27cd8e32-e4e6-4d28-b9c0-2d32c3dfe623)